common/workunit/package.cpp

```diff
@@ -17,6 +17,7 @@
 
 
 #include "platform.h"
+#include "jlib.hpp"
```

Comment:
Agreed, it would have been better to include the required header within jsuperhash.hpp and jhash.hpp.  I'll make this change.